paper.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "PyMedPhys: A community effort to develop an open standard library for Medical Physics in Python"
+tags:
+  - Python
+  - Medical Physics
+  - Radiation Therapy
+  - Diagnostic Imaging
+  - DICOM
+authors:
+  - name: Simon Biggs
+    affiliation: 1
+  - name: Stuart Swerdloff
+    affiliation: 2
+  - name: Matthew Jennings
+    affiliation: 3
+  - name: Phillip Chlap
+    affiliation: "4, 5"
+  - name: Jacob Rembish
+    affiliation: 6
+  - name: Jacob McAloney
+    affiliation: 7
+  - name: Paul King
+    affiliation: 8
+  - name: Rafael Ayala
+    affiliation: 9
+  - name: Matthew Sobolewski
+    affiliation: "7, 10"
+affiliations:
```

Comment:
should I add my name and affiliation if I want to be on the authors list?

Reply:
Yes please! :) Thanks Derek :)

Reply:
Put your name in between the two Jacobs:

![image](https://user-images.githubusercontent.com/6559099/161347795-815d203f-220a-495d-bc05-a443fd1b5e56.png)

And then just tidy up the affiliation numbering and ordering so that it is still in order once you also put the affiliation in.